e2e/src/test/java/io/aiven/kafka/tieredstorage/e2e/LocalSystemSingleBrokerTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2023 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.tieredstorage.e2e;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.kafka.common.TopicIdPartition;
+import org.apache.kafka.common.Uuid;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LocalSystemSingleBrokerTest extends SingleBrokerTest {
+    // File system backend.
+    static final String TS_DATA_SUBDIR_HOST = "ts-data";
+    static final String TS_DATA_DIR_CONTAINER = "/home/appuser/kafka-tiered-storage";
+
+    // File system backend.
+    static Path tempDirTsData;
+
+    @BeforeAll
+    static void init() {
```

Comment:
Should be renamed perhaps since parent has a static method with the same name. Same in other test class

Reply:
Agree, initialization it's a bit confusing. Refactoring it to center it on the Kafka container setup, have a look. Thanks.